src/mtype.d

```diff
@@ -3678,17 +3675,17 @@ public:
             case Tcomplex32:
             case Timaginary32:
             case Tfloat32:
-                fvalue = FLT_EPSILON;
+                fvalue = real_t(FLT_EPSILON);
```

Comment:
Also, I'm probably not making myself clear enough that GDC has a blanket ban on *all* floating point types and literals.  That includes using values from ``math.h`` that are known to the GCC internals at compile-time.  I'm not going to relax this bad either just because the front-end switched to D.